resharper/resharper-unity/src/Yaml/Psi/Modules/UnityExternalFilesModuleProcessor.cs

```diff
@@ -366,7 +366,7 @@ private void OnProjectDirectoryChange(FileSystemChangeDelta delta)
                             projectFilesToAdd.Add(delta.NewPath);
                         }
                     }
-                    else if (delta.NewPath.IsInterestingMeta())
+                    else if (delta.NewPath.IsMeta())
```

Comment:
Does this mean we're indexing ALL meta files now?

Reply:
Yes, are you expecting any performance problem? Meta files are small and for upcoming features we will need guids for differrent assets

Reply:
No, just confirming. Sounds good to me 👍